### DISCUSSION TITLE
posts/KT-%EC%B5%9C%EC%A2%85-%ED%95%A9%EA%B2%A9/

### DISCUSSION THREAD

**Discussion by giscus** [Jihwan98/Jihwan98.github.io#15] [General]
# posts/KT-%EC%B5%9C%EC%A2%85-%ED%95%A9%EA%B2%A9/

🙌 KT 취뽀!!

https://jihwan98.github.io/posts/KT-%EC%B5%9C%EC%A2%85-%ED%95%A9%EA%B2%A9/

**Comment by P-uyoung** [comment]
안녕하세요 :) 혹시 최종 면접 준비는 어떻게 하셨을까요?

**Reply by Jihwan98** [reply]
안녕하세요! 저는 최종 면접으로 따로 준비한 건 없고, 실무 면접 준비랑 동일하게 했어요. 면접 준비는 제가 예전에 했던 프로젝트나 경험들을 정리하는 위주로 했습니다. 그러면서 제가 어필하고 싶은 부분들을 잘 생각해두고, 예상 질문들 찾아보면서 답변 준비했던 것 같아요. 면접 때 생각해보면 결국엔 준비했던 질문들이 그대로 나오는 경우는 없었던 것 같구요(물론 매우 뻔한 클리셰 질문들은 나왔던 것 같긴합니다) 제 경험들을 근거로 질문에 솔직하게 답변하다보니 좋은 결과 있었던 것 같습니다! 다른 회사 면접 경험이 없어서 잘 모르겠지만 제가 생각하기에 KT 면접은 편안한 분위기이기도 하고 질문 난이도도 빡센(?) 느낌은 아닌 것 같아요. 면접 준비 잘 하시구 좋은 결과 있길 바라겠습니다 :)

**Reply by P-uyoung** [reply]
답변 감사드립니당~!

**Comment by GH-Door** [comment]
안녕하세요!
KT 에이블스쿨 취업 관련 질문드립니다.
현재 KT 에이블스쿨 AI 트랙 7기에 합격했는데 다른 부트캠프랑 고민 중입니다.
이유는 데이터 분석가, 사이언티스트 직무로써의 커리어를 희망하는데 취업 연계에 성공하신 분들의 후기를 보면 SW개발자의 직무가 많더라고요, 혹시 AI 관련 직무로도 취업하신 분들이 있으신가요?

**Reply by Jihwan98** [reply]
안녕하세요!

우선, 제가 아는 바로는 현재 에이블스쿨 연계 채용은 종료된 것으로 알고 있습니다. 과거 연계 채용이 진행되었을 때는 주로 SW개발 직군 위주로 채용이 이루어졌으며, 일부 기수에서는 IT 직군 공채에서 서류 전형을 통과하는 방식으로 연계 채용을 진행한 것으로 알고 있습니다. 만약 현재도 같은 방식으로 연계 채용을 진행하고 있다면 KT 신입 채용공고를 확인하셔서 선택하시면 좋을 것 같습니다.

AI 관련 직무로의 취업에 대한 질문을 주셨는데, 말씀하신 것 처럼 데이터 분석가, 사이언티스트 직무를 말씀하시는 것이라면 저희 회사에서 석사채용을 하고있는 것으로 알고있습니다. 다만, 최근 회사가 AI 분야를 강조하고 있는 만큼, 꼭 해당 채용으로 들어오지 않더라도 관련 업무를 담당할 기회가 생길 가능성이 있습니다. 또한 데이터 분석가, 사이언티스트 직무가 아니더라도 AI 관련된 업무는 회사 전반적인 방향성이라 AI 관련 업무를 진행할 기회가 있을 것 같습니다.

채용 공고를 확인하시면서 좋은 결정하시길 바랍니다. :)
